functions/helpers/hrmDataManipulation.private.ts

```diff
@@ -16,9 +16,9 @@
 
 import { get } from 'lodash';
 // eslint-disable-next-line prettier/prettier
-import type { LexMemory } from '../channelCapture/lexClient.private';
 import type { OneToManyConfigSpec } from './insightsService.private';
 
+type LexMemory = { [q: string]: string | number };
```

Comment:
Is it worth adding a comment to mention this also exists in `insightsService`, in case we want to keep them in sync?